course-content/21_getting_data_with_pandas.qmd

```diff
@@ -0,0 +1,427 @@
+---
+title: "Getting Data With Pandas"
+teaching: 45
+exercises: 10
+questions:
+- "Fill this in!"
+objectives:
+- "Fill this in too!"
+keypoints:
+- "Same verse!"
+---
+
+<center>
+    <img src="images/getting-data-with-pandas/cute_panda.jpeg" 
+    width=600 style="margin:auto"/>
+    <p style="text-align: center">
+        Pandas!
+    </p>
+</center>
+
+## Introduction to Pandas
+
+Pandas is one of the most widely-used Python libraries for statistics, and excels at processing tabular data. If you have ever had any experience with R, it is modeled extensively on the dataframes used there. In this section we aim to learn about the fundamental objects used in pandas, bring our own data into these objects, and then view basic information about it. 
```

Comment:
Maybe mention pandas is a portmanteau / mashup of words panel and data. That might make it easier for people to remember what it is and why its helpful (and differentiate its value from numpy, another mashup).

Reply:
I did not actually know that's where the name came from! Very cool.